templates/translations/reminder.html.twig

```diff
@@ -0,0 +1,20 @@
+{% block _translation %}
+
+    {% trans_default_domain 'reminder' %}
+    {# Help the translation extractor #}
+
+    {# Statuses #}
+    {{ 1|trans }}
+    {{ 2|trans }}
+    {{ 3|trans }}
```

Comment:
I'm not sure what this does?

Reply:
Used Twig `constant(ReminderStatus::PENDING)` etc. instead.